app/js/views/navigation-menu.jsx

```diff
@@ -20,19 +20,67 @@ export default class NavigationMenu extends React.Component {
   render() {
     let route = location.hash.substr(1).split('/').shift();
     let menuNodes = [
-      ['services', 'Home'],
-      ['themes', 'Themes'],
-      ['mr-fox', 'Mr. Fox']
+      {
+        id: 'services',
+        label: 'Home',
+        icon: (
+          <svg xmlns="http://www.w3.org/2000/svg" height="30" width="30">
+            <path d="M3 27h8.256v-9.39h7.488V27H27V11.435L15 3 3 11.435V27z"
+                  fill="#fff"/>
+          </svg>
```

Comment:
@azasypkin: I embedded the SVG files into the JSX to be able to style the `path`s using CSS (the icons are white by default and blue when active). I'd like your opinion on this approach. I don't really like it but I can't find of any other way to style icon in CSS.